cypress/integration/folder-contents.js

```diff
@@ -118,5 +112,11 @@ if (Cypress.env('API') !== 'guillotina') {
         ' My Child',
       );
     });
+
+    it('Expiration date must not been set by default', () => {
+      cy.get('.item.dropdown.right').click();
+      cy.get('.icon.Expiration.date').click();
+      cy.get('tbody>tr>td').eq(6).should('have.text', 'None');
```

Comment:
@iFlameing this selector might be a bit too broad. What if there is another "tbody" element in the page? Could you use a more specific selector please?